### PR TITLE
feat: avatar component

### PR DESCRIPTION
**Pull request by tanishk15sharma**
I have added images for Avatar Component :
1)Normal - avatar
2)Round - avatar round
3)Size - sm lg xl

### CODE REVIEW

File: style/docs.css [hunk 1]
```diff
@@ -127,6 +130,7 @@ ul {
   margin-top: 1rem;
   margin-bottom: 2rem;
 }
+
```
Comment (ScoobyDubeyDo) [inline style/docs.css:133]: what is this "+" for?

**Review by ajit-sharmaGH — APPROVED** [review]
Hello I went through your code, it looks cool to merge 😎 !

File: style/docs.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply style/docs.css:133]: That is added by Github, just suggesting us to add an extra line at the end of each file

File: style/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.lg {
+  width: 110px;
+  height: 110px;
+}
+
+.xl {
+  width: 150px;
+  height: 150px;
+}
```
Comment (JantuDeb) [inline style/components/avatar.css:25]: The naming convention is clean.

**Review by JantuDeb — APPROVED** [review]
You can merge this branch.

**Pull request merged**